homeassistant/components/wallbox/switch.py

```diff
@@ -54,11 +54,14 @@ def __init__(
     @property
     def available(self) -> bool:
         """Return the availability of the switch."""
-        return self.coordinator.data[CHARGER_STATUS_DESCRIPTION_KEY] in {
-            ChargerStatus.CHARGING,
-            ChargerStatus.DISCHARGING,
-            ChargerStatus.PAUSED,
-            ChargerStatus.SCHEDULED,
+        return self.coordinator.data[CHARGER_STATUS_DESCRIPTION_KEY] not in {
+            ChargerStatus.UNKNOWN,
+            ChargerStatus.UPDATING,
+            ChargerStatus.ERROR,
+            ChargerStatus.LOCKED,
+            ChargerStatus.LOCKED_CAR_CONNECTED,
+            ChargerStatus.DISCONNECTED,
+            ChargerStatus.READY,
         }
```

Comment:
```suggestion
        return super().available and self.coordinator.data[CHARGER_STATUS_DESCRIPTION_KEY] not in {
            ChargerStatus.UNKNOWN,
            ChargerStatus.UPDATING,
            ChargerStatus.ERROR,
            ChargerStatus.LOCKED,
            ChargerStatus.LOCKED_CAR_CONNECTED,
            ChargerStatus.DISCONNECTED,
            ChargerStatus.READY,
        }
```

Reply:
Should also check that coordinator is available like the other entities are doing automatically